satellite/console/service.go

```diff
@@ -187,14 +187,30 @@ func (s *Service) AddNewPaymentMethod(ctx context.Context, paymentMethodToken st
 		return nil, err
 	}
 
+	var customerID []byte
 	userPayments, err := s.store.UserPayments().Get(ctx, authorization.User.ID)
 	if err != nil {
-		return nil, err
+		if err != sql.ErrNoRows {
+			return nil, err
+		}
+
+		cus, err := s.pm.CreateCustomer(ctx, payments.CreateCustomerParams{
+			Email: authorization.User.Email,
+			Name:  authorization.User.FullName,
+		})
+		if err != nil {
+			return nil, err
+		}
+		customerID = cus.ID
+	}
+
+	if userPayments != nil {
```

Comment:
is there any case when userPayments could be nil, when err also nil? If not, would it better to use if/else, instead of if + if?

Reply:
yes, When you create your first payment method and try to reset 'isDefault' you will have err = sql.noRows and userPayments = nil